### PR TITLE
initial push

### PR DESCRIPTION
**Pull request by willieragan**
## Checkpoint Rubric
This is the rubric that your instructor will use to grade your checkpoints. Please do not edit. 

**Checkpoint 1**
- [ ] A CSS file is effectively linked to the HTML file in the Head section: 15 points
- [ ] A link to a CSS reset file is included in the Head section of the HTML file: 15 points 
- [ ] Appropriate use of Semantic Elements: 15 points
- [ ] Effective use of box model: 15 points
- [ ] Proper use of documentation (commenting on code): 15 points
- [ ] Linter passing: 10 points
- [ ] The student demonstrated mastery of HTML/CSS by building a website of professional caliber: 15 points

**Checkpoint 2**
- [ ] Effective use of Flexbox layout: 15 points
- [ ] Responsive site (resizes to fit desktop, tablet, and mobile views): 15 points 
- [ ] Effective use of CSS Grid: 15 points
- [ ] Proper use of documentation (commenting on code): 15 points
- [ ] Site is hosted on github: 15 points
- [ ] Linter passing: 10 points
- [ ] The student demonstrated mastery of HTML/CSS and Responsive Web Design by building a website of professional caliber: 15 points

**Checkpoint 3**
- [ ] Use of HTML: 15 points
- [ ] Use of CSS: 15 points
- [ ] Use of JQuery: 15 points
- [ ] Proper use of documentation (commenting on code): 15 points
- [ ] Linter passing: 10 points
- [ ] The application has a professional look: 15 points
- [ ] The application functions as it should: 15 points

### CODE REVIEW

File: 02week/blog/index.html [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+<!-- saved from url=(0052)file:///C:/Users/DW/Desktop/htmlcheatsheet/blog.html -->
+<html <head=""><head><meta http-equiv="Content-Type" content="text/html; charset=windows-1252"><title>My Blog</title>
+    <meta name="description" content="awesome blog by will ragan">
+    <meta name="keywords" content="web design blog, web dev blog, Will Ragan">
+  </head>
+  <body><header id="Main-header">
+    <h1>My Blog</h1>
+  </header>
+
+  <a href="file:///C:/Users/DW/Desktop/htmlcheatsheet/index.html">Go to index</a>
```
Comment (situlillian) [inline 02week/blog/index.html:11]: Let's check the path on this again; everything is relative so you don't need to include everything from your CL/Users path.. if you want to include another file, you do need to bring it into your project first

**Review by situlillian — APPROVED** [review]
Good use of semantic tags!

I would like to see some styling on future projects if you have time.

Go ahead and merge it.

**Comment by willieragan** [comment]
Okay....I'll be there in 30 minutes... running about 10 minutes behind.

**Comment by willieragan** [comment]
i think i need some assistance doing it...sorry.

**Pull request merged**